middleware/accuracy.js

```diff
@@ -3,24 +3,23 @@
  * Accuracy level should be set for each item in the results.
  * The level can be any of the following:
  *  - point
- *  - interpolated
+ *  - interpolated (not currently used)
```

Comment:
huh funny. We do return `match_type: 'interpolated'` for interpolated results, but I guess the plan was to return `accuracy: 'interpolated'` too? It kinda makes sense, but also feels redundant, should we still do that?

Reply:
I'm not sure about the history of it, I just noticed that this was dead code and so I commented it out for clarity.